static/js/stream_data.js

```diff
@@ -581,6 +581,14 @@ export function get_invite_only(stream_name) {
     return sub.invite_only;
 }
 
+export function get_web_public_only(stream_name) {
+    const sub = get_sub(stream_name);
+    if (sub === undefined) {
+        return false;
+    }
+    return sub.is_web_public;
+}
```

Comment:
Given that we have `is_web_public`, it seems a cleaner name would be `is_web_public_by_stream_name`; I'll just switch this and merge.

Reply:
(Certainly the "only" in `get_invite_only` is part of the label for the `invite_only` field, not the `get` part).

Reply:
I'll fix this; will also add a few more commits completing this name migration for the module.